### DISCUSSION TITLE
Creating a Windows toast system without third party installations.

### DISCUSSION THREAD

**Discussion by AshutoshAvadhani** [Windos/BurntToast#94] [General]
Hello Sir, my name is Ashutosh Avadhani. As you know that I have to create a toast system without using any third-party installations. I have created a PowerShell script that can display a toast notification but I wanted a way to import this into a windows service. Can this be done where my existing application consume the new said service to display toast notifications?

Thank You for you time and help.

**Comment by Windos** [comment]
Hey there @AshutoshAvadhani, what're you trying to achieve with this "Windows Service"?

When generating your toast notification from a PowerShell script, all you have to do is run the script. You could do this by calling powershell and specifying a file to run (e.g. powershell.exe -file toast.ps1).

**Comment by Windos** [comment]
Hello, and thank you for your contribution to BurntToast.

Today marks the official release of [BurntToast v1.0.0](https://github.com/Windos/BurntToast/releases/tag/v1.0.0). As part of this major milestone and to ensure the project starts its next chapter with a clean slate, I am declaring "issue bankruptcy."

This decision is explained in more detail in the official announcement post: [Ten Years Toasting: BurntToast Hits the Big v1!](https://toastit.dev/2025/07/22/burnttoast-v1/)

As a result, I am closing all outstanding issues and pull requests (and discussions), including this one. Please do not interpret this as a lack of interest in the problem, suggestion, or topic you've raised. This is a necessary step to clear a very old backlog and focus development efforts on the current state of the module.

**If this topic is still relevant to you:**

1.  Please **open a new discussion**. You are welcome to link back to this one for historical context.

I hope this reset will allow me to be more responsive to community feedback going forward.

Thank you for your understanding and for being part of the BurntToast journey.
